File: Seminar5/model/Phone.java
package model;

public class Phone {

    private String phoneNumber;
    
    public Phone(){

    }

    public Phone(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    @Override
    public String toString() {
        return phoneNumber;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

}

File: Seminar7/Model/Phone.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Seminar7.Model
{
    public class Phone
    {
        private string _phoneNumber;

        public Phone()
        {

        }
        public Phone(string phoneNumber)
        {
            _phoneNumber = phoneNumber;
        }

        public string PhoneNumber { get { return _phoneNumber; } set { _phoneNumber = value; } }

        public override string ToString()
        {
            return _phoneNumber;
        }
    }
}
